Chessington.GameEngine/Pieces/Queen.cs

```diff
@@ -10,7 +10,10 @@ public Queen(Player player)
 
         public override IEnumerable<Square> GetAvailableMoves(Board board)
         {
-            return Enumerable.Empty<Square>();
+            var lateralMoves = GetAvailableLateralMoves(board);
+            var diagonalMoves = GetAvailableDiagonalMoves(board);
+
+            return lateralMoves.Union(diagonalMoves);
```

Comment:
Is there any chance of the same moves being in both lists? If not you could just use `Concat` 🙂

Reply:
There shouldn't be any chance of that happening so I've changed that in fbf9e81 🙂